packages/react-topology/src/components/groups/DefaultGroupExpanded.tsx

```diff
@@ -61,14 +69,15 @@ export function computeLabelLocation(points: PointWithSize[]): PointWithSize {
 }
 
 const DefaultGroupExpanded: React.FC<DefaultGroupExpandedProps> = ({
+  className,
   element,
   collapsible,
   selected,
   onSelect,
   hover,
   label,
   secondaryLabel,
-  showLabel,
+  showLabel = true,
```

Comment:
is this changing a default behavior? Could that be unwelcome/breaking for some products

Reply:
Previous change changed it so this should be fixing anything that got inadvertently changed before.